invenio/modules/upgrader/upgrades/invenio_2015_07_14_innodb.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Upgrades MySQL database to InnoDB."""
+
+from invenio.ext.sqlalchemy import db
+
+from invenio.legacy.dbquery import run_sql
+
+from invenio.modules.upgrader.api import op
+
+
+depends_on = ['invenio_2015_03_03_tag_value.py']
+
+
+def info():
+    """Return upgrade recipe information."""
+    return "Upgrades MySQL database to InnoDB."
+
+
+def do_upgrade():
+    """Carry out the upgrade."""
+    from invenio.config import CFG_DATABASE_NAME, CFG_DATABASE_TYPE
```

Comment:
can I do this safely at this point? or is something going to explode when using the entire flask app when still having the wrong tables in place?